scrapers/scrape_be.sh

```diff
@@ -8,32 +8,65 @@ sc.timestamp()
-                        </tr>
-        </tbody>
-    </table>
-</div></div>
-<a name="middlePar_textbild_4bad" style="visibility:hidden"></a><div class="textBild floatingComponent section"><p>(Stand: 25. März 2020)</p>
+<div class="tabelle floatingComponent section">
+    <h2>Corona Erkrankungen im Kanton Bern</h2>
+    <div class="scrolltable">
+        <table cellspacing="0" summary="Laufend aktualisierte Zahlen zu den Corona-Erkrankungen im Kanton Bern">
+            <colgroup>
+                <col width="10%">
+                <col width="10%">
+                <col width="10%">
+                <col width="10%">
+                <col width="10%">
+                <col width="10%">
+                <col width="10%">
+            </colgroup>
+            <thead>
+                <tr>
+                    <th id="th_top_5147_1A"><strong>Datum</strong></th>
+                    <th id="th_top_5147_2A"><strong>Fälle</strong><br> positiv
+                    </th>
+                    <th id="th_top_5147_3A">Im<br>
+                        <strong>Spital</strong><br> gesamt
+                    </th>
+                    <th id="th_top_5147_4A">Davon<br> normale
+                        <br>
+                        <strong>Betten-<br>
+ station</strong></th>
+                    <th id="th_top_5147_5A">Davon<br>
+                        <strong>Intensiv-<br>
+ station</strong><br> gesamt
+                    </th>
+                    <th id="th_top_5147_6A">Davon<br> Intensiv-
+                        <br> pflege
+                        <br>
+                        <strong>beatmet</strong></th>
+                    <th id="th_top_5147_7A"><strong>Todes-<br>
+ fälle</strong></th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td headers="th_top_5147_1A">30.03.2020</td>
+                    <td headers="th_top_5147_2A">826</td>
+                    <td headers="th_top_5147_3A">112</td>
+                    <td headers="th_top_5147_4A">91</td>
+                    <td headers="th_top_5147_5A">21</td>
+                    <td headers="th_top_5147_6A">17</td>
+                    <td headers="th_top_5147_7A">13</td>
+                </tr>
+            </tbody>
+        </table>
+    </div>
+</div>
 """
 
-print('Date and time:', sc.find(r'Stand: (.+)\)', d))
 # This isn't really the best method, but it works for now.
-print('Confirmed cases:', sc.find(r'<td .*<strong>([0-9]+)<', d))
-print('Deaths:', sc.find(r'<td[^<>]*>([0-9]+)</td>', d))
+print('Date and time:', sc.find(r'<td headers="th_top_5147_1A">(.+)<', d))
+print('Confirmed cases:', sc.find(r'<td headers="th_top_5147_2A">([0-9]+)<', d))
+print('Hospitalized:', sc.find(r'<td headers="th_top_5147_3A">([0-9]+)<', d))
+print('ICU:', sc.find(r'<td headers="th_top_5147_5A">([0-9]+)<', d))
+print('Vent:', sc.find(r'<td headers="th_top_5147_6A">([0-9]+)<', d))
+print('Deaths:', sc.find(r'<td headers="th_top_5147_7A">([0-9]+)<', d))
```

Comment:
I don't trust, that these ids like `th_top_5147_1A` are stable, so I wouldn't match exactly for them.

Note that there was another PR in the meantime for the same scraper: #329, it's matching seems a bit better, so I'll use their solution.